### PR TITLE
Made changes related to remove functionality.

### PR DESCRIPTION
**Pull request by kuldipem**

### CODE REVIEW

File: public/modules/articles/views/view-article.client.view.html [hunk 1]
```diff
@@ -6,7 +6,7 @@ <h1 data-ng-bind="article.title"></h1>
 		<a class="btn btn-primary" href="/#!/articles/{{article._id}}/edit">
 			<i class="glyphicon glyphicon-edit"></i>
 		</a>
-		<a class="btn btn-primary" data-ng-click="remove();">
+		<a class="btn btn-danger" onclick="return confirm('Are you sure want to remove ?');" data-ng-click="remove();">
```
Comment (ilanbiala) [inline public/modules/articles/views/view-article.client.view.html]: Should be `ng-click` not `onclick`, and why return it? Also, don't use an ugly confirm dialog.

**Comment by ilanbiala** [comment]
@lirantal @NeverOddOrEven I think most people would be prompted before deleting something to confirm. I like the idea, but I think some things need to be fixed. Thoughts?

**Comment by NeverOddOrEven** [comment]
There's a conversation in the generator here kulpidem/generator-meanjs#53.

**Comment by NeverOddOrEven** [comment]
Here's the direct link https://github.com/meanjs/generator-meanjs/pull/53

**Comment by lirantal** [comment]
@ilanbiala you're absolutely correct - the idea is good and I'm up for adding it but the coding convention there is primitive. @kuldipem please update to use proper angular/javascript code to confirm deletions.

**Comment by kuldipem** [comment]
@lirantal @NeverOddOrEven  , I have just updated onclick => ng-click, is it ok ?

**Comment by lirantal** [comment]
I'm not the authority for confirming that as my strong side isn't AngularJS atm but I think there are probably cleaner, and better ways to create a confirmation dialog than to do 'return confirm()' which is the default browser window function.

File: public/modules/articles/views/view-article.client.view.html [hunk 2]
```diff
@@ -6,7 +6,7 @@ <h1 data-ng-bind="article.title"></h1>
 		<a class="btn btn-primary" href="/#!/articles/{{article._id}}/edit">
 			<i class="glyphicon glyphicon-edit"></i>
 		</a>
-		<a class="btn btn-primary" data-ng-click="remove();">
+		<a class="btn btn-danger" ng-click="return confirm('Are you sure want to remove ?');" data-ng-click="remove();">
```
Comment (ilanbiala) [inline public/modules/articles/views/view-article.client.view.html]: You can't have `ng-click` and `data-ng-click`...one will override the other. Also, stop using confirm(), it is not pretty...Keep all the logic in the controller.

**Comment by lirantal** [comment]
@kuldipem since @ilanbiala's comments haven't been addressed I'm closing this PR.
You or anyone else are welcome to open a new PR for 0.4.0 branch to address this problem.

**Pull request closed without merging**